scipy/sparse/linalg/eigen/_svds.py

```diff
@@ -108,20 +104,20 @@ def svds(A, k=6, ncv=None, tol=0, which='LM', v0=None,
     -------
     u : ndarray, shape=(M, k)
         Unitary matrix having left singular vectors as columns.
-        If `return_singular_vectors` is "vh", this variable is not computed,
-        and ``None`` is returned instead.
+        If `return_singular_vectors` is ``"vh"``, this variable is not
+        computed, and ``None`` is returned instead.
     s : ndarray, shape=(k,)
         The singular values.
-    vt : ndarray, shape=(k, n)
+    vh : ndarray, shape=(k, N)
         Unitary matrix having right singular vectors as rows.
-        If `return_singular_vectors` is "u", this variable is not computed,
-        and None is returned instead.
-
+        If `return_singular_vectors` is ``"u"``, this variable is not computed,
+        and ``None`` is returned instead.
 
     Notes
     -----
     This is a naive implementation using ARPACK or LOBPCG as an eigensolver
-    on A.H * A or A * A.H, depending on which one is more efficient.
+    on ``A.conj().T @ A`` or ``A @ A.conj().T``, depending on which one is more
+    efficient.
```

Comment:
"A.H" is neither code nor math. I'd prefer to make it one of the two. (Of course, I'd prefer math, but use of LaTeX seems too controversial... even though it would be more readable, even in the console.)